src/templates/static/mt5_signals.jsx

```diff
@@ -0,0 +1,238 @@
+                                        {it.L('Benefits of being an MT5 provider')}
+                                    </h2>
+                                    <div className='fill-bg-color gr-padding-30'>
+                                        <div className='container section'>
+                                            <div className='gr-padding-10 facts'>
+                                                <div className='gr-row no-padding'>
+                                                    <Box>
+                                                        <BoxInner className='border-right-top' image='easy_setup' text={it.L('Easy to set up – no installation required.')} />
+                                                        <BoxInner className='border-right-top' image='additional_income' text={it.L('Additional income stream from monthly subscriptions')} />
+                                                        <BoxInner className='border-right-top' image='automated_copy_trade' text={it.L('Automated copying – no extra work from you')} />
+                                                    </Box>
+                                                </div>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                            <div className='gr-padding-30' >
+                                <div className='container gr-padding-30 gr-child'>
+                                    <h2 className='center-text'>
+                                        {it.L('How to register as a signal provider')}
+                                    </h2>
+                                    <div className='gr-row gr-padding-30'>
+                                        <div className='gr-4 gr-12-m gr-padding-10 gr-child'>
+                                            <img className='responsive' src={it.url_for('images/pages/mt5_signals/signal_provider_register_1.png')} />
+                                            <ol>
+                                                <li>{it.L('Go to the MQL5 signals showcase page and click on the [_1]Create signal[_2] button.', '<strong>', '</strong>')}</li>
+                                            </ol>
+                                        </div>
+                                        <div className='gr-4 gr-12-m gr-padding-10 gr-child'>
+                                            <img className='responsive' src={it.url_for('images/pages/mt5_signals/signal_provider_register_2.png')} />
+                                            <ol start='2'>
+                                                <li>{it.L('Complete the form with your Deriv MT5 account credentials. For the [_1]Broker[_2] field, add [_1]Deriv-Server[_2]  as your broker or [_1]Deriv-Demo[_2] if your signal is for demo accounts only.', '<strong>', '</strong>')}</li>
```

Comment:
IMO we shouldn't be saying Deriv MT5 here as we don't use that in binary for now
```suggestion
                                                <li>{it.L('Complete the form with your MT5 account credentials. For the [_1]Broker[_2] field, add [_1]Deriv-Server[_2]  as your broker or [_1]Deriv-Demo[_2] if your signal is for demo accounts only.', '<strong>', '</strong>')}</li>
```

Reply:
Okay, will tell the design team to update this as well